hapi/contact_lists.py

```diff
@@ -15,13 +15,28 @@ def __init__(self, *args, **kwargs):
         super(ContactListsClient, self).__init__(*args, **kwargs)
         self.log = logging_helper.get_log('hapi.contact_lists')
 
-    def _get_path(self, subpath):
-        return 'contacts/v%s/%s' % (self.options.get('version') or CONTACT_LISTS_API_VERSION, subpath)
+    def _get_path(self, method):
+        return 'contacts/v%s/%s' % (self.options.get('version') or CONTACT_LISTS_API_VERSION, method)
 
     def get_contact_lists(self, **options):
+        """ Returns all of the contact lists """
         return self._call('lists', method='GET', **options)
 
-    def add_contact_to_a_list(self, list_id, data=None, **options):
+    def add_contact_to_a_list(self, list_id, vids, data=None, **options):
+        """ Adds a list of contact vids to the specified list. """
         data = data or {}
+        data['vids'] = vids
         return self._call('lists/{list_id}/add'.format(list_id=list_id),
                           data=data, method='POST', **options)
+
+    def create_contact_list(self, list_name, portal_id, dynamic=True, data=None, **options):
+        """ Creates a contact list with given list_name on the given portal_id. """
+        data = data or {}
+        data['name'] = list_name
+        data['portal_id'] = portal_id
+        data['dynamic'] = dynamic
+        return self._call('lists', data=data, method='POST', **options)
+
+    def delete_contact_list(self, list_id, **options):
```

Comment:
Name not consistent: "delete_contact_list" VS "delete_a_contact"
I'm following the name on the documentation, so I'll suggest using delete_a_contact_list here.

Also, any reason we have **options for this method, but not for delete_a_contact() ?